tasks/src/main/java/com/example/task/CreateHttpTaskWithToken.java

```diff
@@ -32,24 +32,24 @@ public static void main(String[] args) throws Exception {
     String projectId = System.getenv("PROJECT_ID");
     String queueName = System.getenv("QUEUE_ID");
     String location = System.getenv("LOCATION_ID");
-    String url = System.getenv("URL");
 
     // Instantiates a client.
     try (CloudTasksClient client = CloudTasksClient.create()) {
       // Variables provided by the system variables.
       // projectId = "my-project-id";
       // queueName = "my-queue";
       // location = "us-central1";
-      // url = "https://example.com/taskhandler";
-      String payload = "hello";
+      String email = args[0]; // Cloud IAM service account
+      String url =
```

Comment:
Should we suggest changing this?

Reply:
Since this is a snippet. I have updated with the sample without a main and highlighted the parameters.